### PR TITLE
Sticky adblock banner fixes

### PR DESCRIPTION
**Pull request by Calanthe**
This PR introduces few changes to the sticky ad-block banner:
- A proper data-attribute added to the links
- New test variant added - with different link (https://membership.theguardian.com/join-challenger instead of https://membership.theguardian.com/about/supporter)
- The whole banner is now clickable - added a link which lays on top of the banner

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/adblock-sticky-banner.js [hunk 1]
```diff
@@ -35,35 +54,26 @@ define([
...
+                id: 'variant',
+                test: function () {
+                    new AdblockBanner(variations[_.random(variations.length - 1)]).show();
+                }
+            },
+            {
+                id: 'challenger',
+                test: function () {
+                    variations[0].supporterLink = 'https://membership.theguardian.com/join-challenger?INTCMP=ADBLOCK_BANNER_MONBIOT';
+                    variations[1].supporterLink = 'https://membership.theguardian.com/join-challenger?INTCMP=ADBLOCK_BANNER_MUIR';
```
Comment (davidrapson) [inline static/src/javascripts/projects/common/modules/experiments/tests/adblock-sticky-banner.js]: `/join-challenger` is a temporary URL and is about to be retired. Can you just direct to the homepage, as you'll see the same page.

File: static/src/javascripts/projects/common/modules/adblock-banner.js [hunk 2]
```diff
@@ -45,6 +47,7 @@ define([
 
         fastdom.write(function () {
             $('.top-banner-ad-container--desktop').after(bannerTmpl);
+            $('.js-adblock-link').addClass('adblock-sticky__link');
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/adblock-banner.js]: why add this in JS? why not just have it in the html source?

**Comment by uplne** [comment]
:+1:

**Comment by Calanthe** [comment]
@sndrs I already fixed that. Maybe that's why it was removed ;)

**Comment by sndrs** [comment]
@Calanthe that's really weird, i could still see it – fair enough though!

**Comment by uplne** [comment]
@sndrs I've checked it before and it was already set as outdated (changed) so something weird is happening with github

**Pull request merged**